includes/Post/HandleFinalDataOfCreatePost.php

```diff
@@ -159,7 +159,26 @@
 		echo "<br><br><h3 style = 'color:white ;'> $post Post  already exists .</h3>";
 		die ();
 	}
-
+	
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->